consumer/README.md

```diff
@@ -1,25 +1,55 @@
 # Documentation for consumers of Pipeline Service
 
-This folder holds all the files required to consume Pipeline Service when the service is
-operated by a third party.
+This folder holds all the files required to consume Pipeline Service when the service is operated by a third party.
 
 ## Requirements
 
 * Access to a kcp workspace.
-* Path of the kcp workspace in which the service is deployed (e.g. `root:pipeline-service`).
-This path is provided by the service operators.
+* The following information from the service provider:
+  * Path of the kcp workspace in which the service is deployed (e.g. `root:pipeline-service`).
+  * Name of the APIExport (e.g. `kubernetes`).
 
-## How to bind to the service
+## Quick start guide
 
-* Login on your kcp workspace.
-* Run `./bind_to.sh root:path:to:the:service` to bind to a Pipeline Service provider. The
-exact path will be provided by the service provider.
+This section explains how to use the provided scripts to quickly setup a workspace to use a service provided by an operator of Pipeline Service.
 
-## How to test the service
+If you prefer to do it yourself, or something is not quite working the way it should, see the `Manual setup` section to be walked through the procedure.
 
-* Login on your kcp workspace.
-* Run `./run_test_workload.sh`.
+### Binding to the service
 
-## How to connect a repository to Pipeline as Code
+* Login to your kcp workspace (e.g. `~`).
+* Run `./bind.sh --exportname 'kubernetes' --from 'root:path:to:the:service' --to '~:pipeline-service'`
+  * A new `pipeline-service` workspace will be created under `~`.
+  * The binding to the Pipeline Service provider will be created in the `~:pipeline-service` workspace.
+  * A test will automatically be run to validate that `pipelineruns` get scheduled in the `run to validate that `pipelineruns` get scheduled.` workspace.
```

Comment:
Seems like there is a typo in this sentence.

Reply:
Fixed.